src/Terminal.php

```diff
@@ -4,163 +4,120 @@
 
 /**
  * @author Michael Woodward <mikeymike.mw@gmail.com>
+ * @author Aydin Hassan <aydin@hotmail.co.uk>
  */
-class Terminal implements TerminalInterface
+interface Terminal
```

Comment:
So I've rename `TerminalInterface` to `Terminal` and `Terminal` to `UnixTerminal`. I know we only have 1 terminal but we might add a windows one in the future if the terminal gets a bit better. What do you think?